### PR TITLE
Add support for LiteGraph to convert to classes

### PR DESCRIPTION
**Pull request by webfiltered**
Adds downstream support for litegraph.js to convert to using proper ES6 classes.

Much extension testing required.  =/

### CODE REVIEW

File: src/extensions/core/noteNode.ts [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
       bgcolor = LGraphCanvas.node_colors.yellow.bgcolor
       groupcolor = LGraphCanvas.node_colors.yellow.groupcolor
-      properties: { text: string }
-      serialize_widgets: boolean
       isVirtualNode: boolean
       collapsable: boolean
       title_mode: number
 
       constructor() {
+        super()
```
Comment (pythongosssss) [inline src/extensions/core/noteNode.ts]: This needs to pass the title

File: src/extensions/core/rerouteNode.ts [hunk 2]
```diff
@@ -11,11 +11,12 @@ app.registerExtension({
       __outputType?: string
     }
 
-    class RerouteNode {
+    class RerouteNode extends LGraphNode {
       static category: string | undefined
       static defaultVisibility = false
 
       constructor() {
+        super()
```
Comment (pythongosssss) [inline src/extensions/core/rerouteNode.ts]: This needs to pass the title

File: src/extensions/core/widgetInputs.ts [hunk 3]
```diff
@@ -13,11 +13,12 @@ const TARGET = Symbol() // Used for reroutes to specify the real target widget
 interface PrimitiveNode extends LGraphNode {}
 
 const replacePropertyName = 'Run widget replace on values'
-class PrimitiveNode {
+class PrimitiveNode extends LGraphNode {
   controlValues: any[]
   lastType: string
   static category: string
   constructor() {
+    super()
```
Comment (pythongosssss) [inline src/extensions/core/widgetInputs.ts]: This needs to pass the title

**Comment by webfiltered** [comment]
*facepalm*  Thank you!  I even saw 'Unnamed value' and looked at the title - then utterly failed to connect the dots.

File: src/extensions/core/widgetInputs.ts (hunk 3 above)
Comment (webfiltered) [reply src/extensions/core/widgetInputs.ts]: *facepalm* Thank you! I even saw 'Unnamed value' and looked at the title - then utterly failed to connect the dots.

**Review by huchenlei — APPROVED** [review]
LGTM!

**Pull request merged**